torchbox/torchbox.com
# Image desaturation

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1388897849)

### Description of Changes Made

- Make all images desaturated by 40%
- Convert avatar images to use the `srcset_image` tag (listing images have some art direction involved so I have left these as picture tags)

Note: this will necessitate all image renditions being recreated, so we may have further problems with the team page listing on staging and production until they are re-created again.

### How to Test

View any page with an image in an actual wagtail build - pattern library images should still render but will not be desaturated.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Featured case study example
![Screenshot 2024-02-26 at 12 23 45](https://github.com/torchbox/torchbox.com/assets/771869/d7b9b17c-e592-443a-9b4b-da7dd9759aa3)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
